meeshkan/serve/mock/response_matcher.py

```diff
@@ -39,7 +39,7 @@ def match_error(self, msg: str, req: Request) -> Response:
     def default_response(self, msg):
```

Comment:
Should this maybe be called `error_response` or something similar or is 501 the default?

Reply:
The only place where it is currently used is the match_error function. So the best option would be to merge that functions together because both are very small.

Reply:
`501` is the default

Reply:
@aby2s agreed, organic evolution produced that